requirements.txt

```diff
@@ -5,55 +5,87 @@ aiosignal==1.3.2
 annotated-types==0.7.0
 anyio==4.7.0
 attrs==24.3.0
+cachetools==5.5.0
 certifi==2024.12.14
 cffi==1.17.1
 charset-normalizer==3.4.0
 cryptography==44.0.0
 dataclasses-json==0.6.7
 Deprecated==1.2.15
 distro==1.9.0
+eval_type_backport==0.2.0
+filelock==3.16.1
+filetype==1.2.0
 frozenlist==1.5.0
+fsspec==2024.10.0
+google-ai-generativelanguage==0.6.10
+google-api-core==2.24.0
+google-api-python-client==2.155.0
+google-auth==2.37.0
+google-auth-httplib2==0.2.0
+google-generativeai==0.8.3
+googleapis-common-protos==1.66.0
+groq==0.13.1
+grpcio==1.68.1
+grpcio-status==1.68.1
 h11==0.14.0
 httpcore==1.0.7
+httplib2==0.22.0
 httpx==0.27.2
 httpx-sse==0.4.0
+huggingface-hub==0.27.0
 idna==3.10
 jiter==0.8.2
 jsonpatch==1.33
+jsonpath-python==1.0.6
 jsonpointer==3.0.0
 langchain==0.3.12
 langchain-community==0.3.12
 langchain-core==0.3.25
+langchain-google-genai==2.0.7
```

Comment:
🌐 Ensure `langchain-google-genai==2.0.7` is required for your application. If not, it can be omitted.